pagination/cursor/README.md

```diff
@@ -0,0 +1,57 @@
+## Cursor-based Pagination
+
+This example shows how to use cursor based pagination. This is a type of pagination that works well with GraphQL as it relies on edges and nodes.
```

Comment:
I don't think that's the phrasing we want, connection based pagination is our model, not something that "works well".